CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # 8.5.0
 ### Features
-* New CSS class `title-sidenav-content` for *ontimize-web-ngx-charts@8.2.0*.
+* New CSS class `title-sidenav-content` for *ontimize-web-ngx-charts@8.2.0* ([b275a17](https://github.com/OntimizeWeb/ontimize-web-ngx-theming/commit/b275a17)) Closes [#105](https://github.com/OntimizeWeb/ontimize-web-ngx-charts/issues/105)
+* Fixed buttons style on Implatform Fashion theme on dark mode ([9978321](https://github.com/OntimizeWeb/ontimize-web-ngx-theming/commit/9978321)) Closes [#81](https://github.com/OntimizeWeb/ontimize-web-ngx-theming/issues/81)
+* Added background levels `level-0`, `level-04`, `level-06`, `level-08` and `level-1` on dark mode backgrounds. New `level-04` improved on dark and light mode. Improved background and foreground functions on `ontimize-theme-styles.scss` and `implatform-fashion-theme-styles.scss` ([f329cad](https://github.com/OntimizeWeb/ontimize-web-ngx-theming/commit/f329cad)) Closes [#81](https://github.com/OntimizeWeb/ontimize-web-ngx-theming/issues/81)
+* Fixed checkbox background on `implatform-fashion-default-theme.scss` ([4a9e2a0](https://github.com/OntimizeWeb/ontimize-web-ngx-theming/commit/4a9e2a0))
+* Changed theme file structure ([cb5678b](https://github.com/OntimizeWeb/ontimize-web-ngx-theming/commit/cb5678b)) Closes [#95](https://github.com/OntimizeWeb/ontimize-web-ngx-theming/issues/95)
```

Comment:
Does this affect the user? would change the imports? If so, I would go to the breaking changes section, otherwise I don't know if it makes sense to add it